pending/17-handlebars.pod

```diff
@@ -0,0 +1,84 @@
+=head1 Dancer2::Template::Handlebars
+
+I know that our editor-in-chief this year, the amazing L<Cromedome|https://metacpan.org/author/CROMEDOME>, is hoping
+for some deep wisdom here about why and how I created Dancer2::Template::Handlebars.
+
+He's going to have to get used to disappointment.
+
+I realized a while back that, after nearly 20 years of writing Perl, I didn't have a single module on CPAN.
+It seemed to me that something simple-ish would be a good idea, just so I could work out the workflow, and
+I'd been playing around with some templating using L<Handlebars|https://handlebarsjs.com/> for another project
+of mine. To make it even easier, the underling L<Text::Handlebars> was already present, and our clever colleague
+L<Yanick|https://metacpan.org/author/YANICK> had already written L<Dancer::Template::Handlebars>, from which I
+could crib some code as needed.
+
+All the bits were in place, it was just up to me to learn the bits and pieces of creating, testing, and submitting
+a module.  But why this one?
+
+=head2 Why you should like Handlebars, and use it
+
+Handlebars, like Template Toolkit and many others, can take objects handed to it from Dancer2's template-processing
+system, and insert data into pages, at your direction.  So, if you have a list of people objects with names,
+you can get a list of them:
+
+=begin html
+
+{{#each people}}
+  {{ lastname }}, {{ firstname }}<br>
+{{/each}}
+
+=end html
+
+Handlebars also supports "partials", subtemplates you might want to call, similar in behavior to TT's C<INCLUDE> or C<PROCESS>
+tools. You can even dynamically call a partial by referencing a variable name containing the name of the partial!
+
+What sets Handlebars apart from Template Toolkit is its model of "helpers."  There are a few built-in helpers, like
+C<#each> and C<#with>, which create loop iterators, C<#if> and C<#unless> that create conditionals, and a C<lookup>
+C<log> helper, which provide data access and logging, respectively. But everything else you might want to do is a helper
+you must include or write. Writing them is relatively straightforward JavaScript, like so:
+
+=begin javascript
+
+Handlebars.registerHelper('loud', function (aString) {
+	    return aString.toUpperCase()
+})
+
+=end 
+
+Then in your template,
+
+=begin html
+
+{{loud lastname }}, {{ firstname }}<br>
+
+=end html
+
+And for me, you'd get HOLLOWAY, Ruth.
+
+If you have a bit of HTML your Dancer application is generating, or storing in a database, you can cause those
+to be passed along with a "triple-stache":  C<{{{ generated_html_field }}}>.
```

Comment:
As I'm not all that familiar with Handlebars, I didn't quite get the difference between `{{...}}` and `{{{...}}}`; you alluded to using the latter if you have HTML to pass along, but not quite clearly why that needs to be treated differently.

After a quick Google, I see that `{{...}}` HTML-escapes the values by default, and `{{{...}}}` does not - hence you'd use the latter if you have pre-formed HTML to include.  That all makes sense, but wasn't immediately apparent to a non-Handlebars user, so it might warrant a very brief mention?

On the other hand, you're not trying to write a Handlebars manual here :)

Reply:
I've changed the verbiage a bit here, just to make that clearer. As you point out, I'm not trying to write a manual, but it *is* one of the easy-to-use and handy features of Handlebars, so worth a mention here.